zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -636,8 +633,12 @@ public String getInterpreterPortRange() {
     return getString(ConfVars.ZEPPELIN_INTERPRETER_RPC_PORTRANGE);
   }
 
-  public boolean isWindowsPath(String path){
-    return path.matches("^[A-Za-z]:\\\\.*");
+  public boolean isWindowsCheck(){
+    Boolean wc = SystemUtils.IS_OS_WINDOWS;
+    if(Objects.isNull(wc)) {
```

Comment:
It returns a primitive type. I guess you don't have to check if it's null or not. 

https://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/SystemUtils.html#IS_OS_WINDOWS

Reply:
OK, Thank you.  I will edit and re-upload.